src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -207,6 +244,21 @@ public void auxiliaryPollModeUpdateRequest(Endpoint endpoint, boolean enableAuxi
         emit(STREAM_POLL_ID, makePollTuple(command));
     }
 
+    @Override
+    public void islRulesInstalled(IslReference reference, Endpoint endpoint) {
+        islService.islRulesProcessed(reference, endpoint, IslFsmEvent.ISL_RULE_INSTALLED);
```

Comment:
Transport layer shoul not know anything about `IslFsmEvent`. Better make 2 one line methods inside `islService` and lock all knowlegge about FSM up to service layer.